Traceback (most recent call last):
  File "main.py", line 114, in <module>
    visualize_input_output(inputs, outputs, epoch)
  File "main.py", line 60, in visualize_input_output
    input_image = inputs[index, :image_dim].view(28, 28).cpu().numpy()
IndexError: index 10 is out of bounds for dimension 0 with size 10
